manageiq-providers-ibm_cloud.gemspec

```diff
@@ -18,6 +18,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+
+  spec.add_dependency "sshkey", "~>1.8.0"
```

Comment:
We try to keep these listed alphabetically if you can drop this under the `rest-client` dep

Reply:
I thought this is more a system call. and move it at the top. Just old C habbit. Let me move it down.